thinknimble/tn-spa-bootstrapper
# fix: Make both Vue and React work on Docker out-of-the-box

## What this does

Implements some needed fixes for both React and Vue clients to work right out-of-the-box.
It also unifies behaviour, so no drastic changes are needed.


## How to test

1. Use cookiecutter to create a project, selecting React as the frontend framework.
2. Run `docker compose up client` inside your new project's root.
3. Access http://localhost:8080/ and confirm it works.

4. Use cookiecutter to create a project, selecting Vue as the frontend framework.
5. Run `docker compose up client` inside your new project's root.
6. Access http://localhost:8080/ and confirm it works.

## Fixes
[Docker installs vue stuff when there's no Vue app](https://github.com/thinknimble/tn-spa-bootstrapper/issues/179)
[React client - Vite does not run on docker](https://github.com/thinknimble/tn-spa-bootstrapper/issues/182)
